Source/APIClient/Request.swift

```diff
@@ -18,13 +18,15 @@ public enum HTTPMethod: String {
 public protocol Request {
     static var baseURL: String { get }
     static var version: String { get }
+    static var serverAddress: String? { get }
     static var authToken: String? { get set }
 
     var method: HTTPMethod { get }
     var path: String { get }
     var authenticated: Bool { get }
     var queryParameters: [String: AnyObject]? { get }
     var bodyParameters: [String: AnyObject]? { get }
+    var contentType: String { get }
```

Comment:
My understanding is that this is where you would set, say, `multipart/form-data` for a request case that requires it. To that end, we wouldn't want this to be `static`, right? We'd want certain request cases to return `json` and others to return `multipart/form-data`, `image/jpeg` and so on.

https://en.wikipedia.org/wiki/Media_type

Referencing list from here ☝️.